core-war/src/main/java/org/silverpeas/web/jobdomain/servlets/JobDomainPeasRequestRouter.java

```diff
@@ -469,11 +469,15 @@ public String getDestination(String function, JobDomainPeasSessionController job
       } else if (function.startsWith("domain")) {
         jobDomainSC.setTargetUser(null);
 
-        if (function.startsWith("domainNavigation")) {
+        if (function.startsWith("domainGoTo")) {
+          jobDomainSC.setTargetDomain(request.getParameter("Iddomain"));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Iddomain" 7 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1192)